vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -621,6 +753,36 @@ public void setDataProvider(FetchItemsCallback<T> fetchItems,
                 q -> sizeCallback.apply(q.getFilter().orElse(""))));
     }
 
+    /**
+     * Supplies items lazily with callbacks: the first one fetches the items
+     * based on text filter, offset and limit, the second provides the exact
+     * count of filtered items in the backend. Use this in case getting the
+     * count is cheap and the user benefits from the component showing
+     * immediately the exact size.
+     * <p>
+     * The returned data view object can be used for further configuration, or
+     * later on fetched with {@link #getLazyDataView()}. For using
+     * in-memory data, like {@link java.util.Collection}, use
+     * {@link HasListDataView#setItems(Collection)} instead.
+     * <p>
+     * This method is a shorthand for making a size callback that handles a
+     * partial {@link com.vaadin.data.provider.Query Query} object.
+     *
+     * @param fetchItems
+     *            a callback for fetching items
+     * @param sizeCallback
+     *            a callback for getting the count of items
+     * @return ComboBoxLazyDataView instance for further configuration
+     */
+    // TODO: this methods clashes with
+    // setItems(CallbackDataProvider.FetchCallback<T, F> fetchCallback,
+    //            CallbackDataProvider.CountCallback<T, F> countCallback)
+    public ComboBoxLazyDataView<T> setItems(FetchItemsCallback<T> fetchItems,
+            SerializableFunction<String, Integer> sizeCallback) {
```

Comment:
Probably we'll just have to break this one for unity with other components API.
Or another option to reverse the ordering of the callbacks, but that would still be breaking, so maybe just remove it.

Reply:
This method has been removed to get rid of conflicts. Overload `setItems(FetchCallback<T, F> fetchCallback, CountCallback<T, F> countCallback)` can be used instead